rafthttp/transport.go

```diff
@@ -78,13 +81,24 @@ type Transporter interface {
 	Stop()
 }
 
-type transport struct {
-	roundTripper http.RoundTripper
-	id           types.ID
-	clusterID    types.ID
-	raft         Raft
-	serverStats  *stats.ServerStats
-	leaderStats  *stats.LeaderStats
+// Transport is the implementation of Transporter. It provides the functionality
+// to send raft messages to peers, and receive raft messages from peers.
+// The Handler in Transport needs to serve requests sent to peer URLs of the
+// local member for remote Transport to work well.
+// Transport needs to call Start before calling other functions, and call
```

Comment:
Transport needs to call Start? Or the user of transport needs to call Start? From the code, Start method is called by the user of Transport and never be called by Transport itself.